python/tests/io/test_pyarrow.py

```diff
@@ -796,9 +796,7 @@ def test_projection_add_column(file_int: str) -> None:
     for actual, expected in zip(result_table.columns[2], [None, None, None]):
         assert actual.as_py() == expected
 
-    for actual, expected in zip(
-        result_table.columns[3], [{"lat": None, "lon": None}, {"lat": None, "lon": None}, {"lat": None, "lon": None}]
-    ):
+    for actual, expected in zip(result_table.columns[3], [None, None, None]):
```

Comment:
I don't think that these test cases were correct.

Reply:
This is correct because the `location` field is missing and defaults to `null`. Otherwise, `location` defaults to a non-null value, `Record[None, None]`, which is not correct.